exporter/collector/integrationtest/testdata/fixtures/with_resource_filter_metrics_expect.json

```diff
@@ -0,0 +1,685 @@
+{
+  "createTimeSeriesRequests": [
+    {
+      "timeSeries": [
+        {
+          "metric": {
+            "type": "workload.googleapis.com/testresourcefilter",
+            "labels": {
+              "foo": "bar",
+              "service_instance_id": "abc123",
+              "service_name": "foo-service",
+              "service_namespace": "foo",
+              "telemetry_sdk_language": "java",
+              "telemetry_sdk_version": "1.1.1"
+            }
+          },
+          "resource": {
+            "type": "generic_task",
+            "labels": {
+              "job": "foo-service",
+              "location": "global",
+              "namespace": "foo",
+              "task_id": "abc123"
+            }
+          },
+          "metricKind": "CUMULATIVE",
+          "valueType": "INT64",
+          "points": [
+            {
+              "interval": {
+                "endTime": "1970-01-01T00:00:00Z",
+                "startTime": "1970-01-01T00:00:00Z"
+              },
+              "value": {
+                "int64Value": "253"
+              }
+            }
+          ],
+          "unit": "1"
+        }
+      ]
+    }
+  ],
+  "createMetricDescriptorRequests": [
+    {
+      "metricDescriptor": {
+        "type": "workload.googleapis.com/testresourcefilter",
+        "labels": [
+          {
+            "key": "foo"
```

Comment:
Note that the labels from the monitored resource are NOT in the CMD call as written. I think this should probably be updated.

Reply:
Done